src/ray/core_worker/transport/direct_task_transport.cc

```diff
@@ -248,14 +247,9 @@ void CoreWorkerDirectTaskSubmitter::StealTasksOrReturnWorker(
     return;
   }
 
-  RAY_LOG(DEBUG) << "Beginning to steal work now! Thief is worker: "
```

Comment:
is it because these message to chatty?

Reply:
Mostly because it gives us confusion that work stealing code path is being active

Reply:
I see, you probably want wait for @stephanie-wang as it might be useufl.